manifest.yaml

```diff
@@ -0,0 +1,112 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gitops-plugin
+  namespace: openshift-gitops
+  labels:
+    app: gitops-plugin
+    app.kubernetes.io/component: gitops-plugin
+    app.kubernetes.io/instance: gitops-plugin
+    app.kubernetes.io/part-of: gitops-plugin
+    app.openshift.io/runtime-namespace: openshift-gitops
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: gitops-plugin
+  template:
+    metadata:
+      labels:
+        app: gitops-plugin
+    spec:
+      containers:
+        - name: gitops-plugin
+          image: quay.io/yicai/console-gitops-plugin:v0.0.2
```

Comment:
Do we have a redhat-developer image on quay.io? (If not, I suppose we can change this later when it's ready)

Reply:
Updated with this new quay.io repo url: quay.io/redhat-developer/gitops-console-plugin